hack/lib.bash

```
@@ -0,0 +1,364 @@
+helm::install::kcp() {
+    local kubeconfig="$1"
+    shift 1
+    helm::repo kcp  https://kcp-dev.github.io/helm-charts
+    helm::install "$kubeconfig" \
+        kcp-operator kcp/kcp-operator \
+        "$@"
+}
+
+kubeconfig::hostname() {
+    local kubeconfig="$1"
+    local hostname="$(yq '.clusters[0].cluster.server' "$kubeconfig")"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)